main/2024/CG-06.md

```diff
@@ -22,14 +22,17 @@
     - Link sent to registered attendees.
 - **Location**:
     - Carnegie Mellon University, Pittsburgh, Pennsylvania - [Gates & Hillman Centers](https://maps.app.goo.gl/yG2fozWrGsf4GMua8) 6115.
+
+Here's an interactive campu map of the location of Gates.
```

Comment:
```suggestion
Here's an interactive campus map of the location of Gates.
```

Reply:
oh actually this also seems to be missing the link :)

Reply:
Good catch, fixed.